packages/non-core/facebook/deprecation_notice.js

```diff
@@ -0,0 +1,9 @@
+console.warn(
```

Comment:
Can we make sure these deprecated packages go into `packages/deprecated` instead of `packages/non-core`? I realize I misspoke in our meeting about this, but it seems like a meaningful distinction.